R/oauth-cache.R

```diff
@@ -315,13 +317,14 @@ token_match <- function(candidate, existing, package = "gargle") {
 
   # we need user to OK our discovery or pick from multiple emails
   emails <- extract_email(existing)
-  local_gargle_verbosity("info")
-  gargle_info(c(
+  choice <- cli_menu(
     "The {.pkg {package}} package is requesting access to your Google account.",
-    "Select a pre-authorised account or enter '0' to obtain a new token.",
-    "Press Esc/Ctrl + C to cancel."
-  ))
-  choice <- utils::menu(emails)
+    c(
+      "Select a pre-authorised account or enter '0' to obtain a new token.",
```

Comment:
Should we add an explicit bullet like "Obtain a new token" instead of using 0? (I don't think this code will work now because `cli_menu()` will exit to the top-level if the user enters 0)

Reply:
> I don't think this code will work now because `cli_menu()` will exit to the top-level if the user enters 0

Yes, this is true. I will deal with that :thinking: